drivers/storage/scaleio/tests/scaleio_test.go

```diff
@@ -25,14 +25,22 @@ import (
 
 var (
 	configYAML = []byte(`
+libstorage:
+  service: scaleio
+  integration:
+    volume:
+      operations:
+        mount:
+          preempt: true
 scaleio:
-  endpoint: https://192.168.50.12/api
+  endpoint: https://10.0.0.11/api
   insecure: true
+  thinOrThick: ThinProvisioned
   userName: admin
-  password: Scaleio123
-  systemName: cluster1
-  protectionDomainName: pdomain
-  storagePoolName: pool1
+  password: F00barbaz
+  systemId: 0a79118e5ecc882c
```

Comment:
So where is set that these values are correct?  I don't see it in the CF template (but honestly don't know what I'm looking for). Again, i dislike seeing these, because I think it could trip people up who compile the test binary and choose to run it against their own ScaleIO cluster.

I'm not saying it's wrong though. All of these can (and likely should) be overriden by env vars. I'd just rather not compile in default credentials at all.